src/Features/Core/Portable/SimplifyInterpolation/AbstractSimplifyInterpolationDiagnosticAnalyzer.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Immutable;
+using System.Linq;
```

Comment:
```suggestion
```

Reply:
Can't do unless you'd like me to remove the `.First()` which was paired with the `.Skip(1)`.

Reply:
that's fine.